common/middleware/ensure-authenticated.test.js

```diff
@@ -14,7 +14,7 @@ describe('Authentication middleware', function() {
         session: {
           authExpiry: null,
         },
-        header: sinon.stub().returns(''),
+        get: sinon.stub().returns(''),
```

Comment:
Maybe have this return `undefined` by default in this test suite actually. Then it will force the fallback string.

With this `.returns` it will return a string and the fallback string won't be used.

Reply:
👍

Reply:
In fact, just using `.returns()` will return undefined. Although looking at it, a string would be treated as fasly and the empty string would be used.

Might make it easier to read though not returning the same value as the fallback value in the test mock.